.github/workflows/snapshot-file-change.yml

```diff
@@ -71,11 +72,13 @@ jobs:
           destination: '${{ inputs.destination_prefix }}/pmap-file-change-${{ github.run_id }}-${{ github.run_attempt }}'
```

Comment:
Just curious what if we still use `${{github.run_id}}-${{github.run_attempt}}` as the separator? This way the separator would be unique and has less probability that user accidentally used things like `gh-prefix` in the input pass.
Are there any other specific concerns for doing it this way?

Reply:
Updated to use `gh-prefix` as separator.

Reply:
It's not very obvious that we should/will always have `${{github.run_id}}-${{github.run_attempt}}` before the input path. We _could_ very well use a uuid because we don't really depend on the run id/attempt in the path. On the other hand, we can make a well-known separator very obvious.